storage/rocksdb/ha_rocksdb.cc

```diff
@@ -417,6 +433,8 @@ static char *rocksdb_strict_collation_exceptions;
 static my_bool rocksdb_collect_sst_properties = 1;
 static my_bool rocksdb_force_flush_memtable_now_var = 0;
 static my_bool rocksdb_force_flush_memtable_and_lzero_now_var = 0;
+static my_bool rocksdb_enable_ttl = 1;
```

Comment:
This is going to be nice, MyRocks uses a static global singleton `std::unique_ptr<rocksdb::DBOptions> rocksdb_db_options = rdb_init_rocksdb_db_options();` to get default values for the SYSVARS macros and thus eventually the sysvar values, following on the belief/assumption I made above. So if what you say is true, then they have potential startup init and use ordering issues waiting to happen.